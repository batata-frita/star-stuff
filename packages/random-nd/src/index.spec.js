const randomNd = require('.')

it('should generate a random number based on n dimensions', () => {
  expect(randomNd(1, 2, 3, 4)).toMatchSnapshot()
})
