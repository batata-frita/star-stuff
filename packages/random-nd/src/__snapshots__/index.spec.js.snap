// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should generate a random number based on n dimensions 1`] = `0.8387175938114524`;
